app/config.js

```diff
@@ -8,7 +8,9 @@ const notify = require('./notify');
 const Config = require('electron-config');
 
 // local storage
-const cache = new Config();
+const winCfg = new Config({name: 'winConf'});
```

Comment:
I don't think you want to set a name. That would create a separate config file for just this. Better to have it in the main config.